Exercise_B/index.html

```diff
@@ -1,34 +1,59 @@
 <!DOCTYPE html>
-<html>
+<html class="no-js">
   <head>
     <meta charset="utf-8" />
-    <title>Flower Shop BS</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <!--If you want to use bootstrap properly this is vital (helps browser to measure the page and readjust the information accordingly-->
-
-    <!-- Link and CDN required for bootstrap -->
-
-    <!-- Google Font - Lato -->
-    <link
-      href="https://fonts.googleapis.com/css?family=Lato"
-      rel="stylesheet"
-    />
-    <!-- Custom Stylesheet -->
-    <link rel="stylesheet" href="./assets/style/stylesheet.css" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>FlowerShop</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
+    <link rel="stylesheet" href="assets/style/stylesheet.css" />
   </head>
-
   <body>
-    <!--Header (flower shop logo block element)-->
-    <!--Banner Jumbotron of the image-->
 
-    <!--Four containers with img / text-->
-    <!-- 4 containers -->
+    <div class = "topbar">
+      <img src = "assets/images/logo.png" alt = "logo">
+    
+    </div>
+
+    <div class = "head">
+
+      <div class = "headtitle">
+      <h1 class = "whitetxt heading large">Beautiful Flowers</h1>
+      </div>  
 
-    <!-- Payments Div -->
-    <!-- Consultation Div -->
+      <div class = "lrnbtn">
+        <a class = "green whitetxt medium" href = "index.html">Learn More</a>
+      </div>
 
-    <!-- Delivery Div -->
+    </div>
+    <div class = "footer container ">
+      <div class="row">  
+        <div class = "col-xs-12 col-md-6 col-lg-3 p-3 text-center">
+          <img src = "assets/images/flower-icon.png" alt = "flower-icon">
+          <p class = "float-end p-2">
+          Flowers for every<br />
+          kind of situation
+          </p>
+        </div>
+        <div class = "col-xs-12 col-md-6 col-lg-3 p-3 text-center">
+          <img src = "assets/images/payment-icon.png" alt = "pay">        
+          <p class = "float-end p-2">
+          All major payments<br />
+          kinds accepted
+          </p>   
+        </div>
+        <div class = "col-xs-12 col-md-6 col-lg-3 p-3 text-center">
+          <img src = "assets/images/phone-icon.png" alt = "phone">
+          <p class = "float-end p-2">Free consulation</p>
+        </div>
+        <div class = "col-xs-12 col-md-6 col-lg-3 p-3 text-center">
+          <img src = "assets/images/truck-icon.png" alt = "truck">    
+          <p class = "float-end p-2">City wide delivery</p>
+        </div>
+      </div>
+    </div>
 
-    <!--Scripts required for bootstrap CDN-->
+    <script src="" async defer></script>
   </body>
 </html>

```

Comment:
Very well done here for implementing the bootstrap column system. 
The code that you have developed is good and showcases your understanding of bootstrap.